config/locales/server.en.yml

```diff
@@ -2080,6 +2082,9 @@ en:
 
         If the above link is not clickable, try copying and pasting it into the address bar of your web browser.
 
+    push:
+      body: You have %{count} new notifications.
```

Comment:
It's needed, even if it doesn't make sense for English. Other languages have other pluralization rules and Transifex doesn't detect it as pluralized unless there are `one` and `other` keys in English. Trust me. ;-)